packages/create-single-spa/bin/create-single-spa.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+const yeoman = require('yeoman-environment')
```

Comment:
yeoman will be easier i think to manage than a homegrown CLI thing. A separate npm package called `generator-single-spa` will do the actual generation of all the files and stuff